.github/workflows/generate_utcs.py

```diff
@@ -0,0 +1,73 @@
+import json
+import os
+
+
+def add_effected_actions(template: str, info: dict) -> str:
+    template += "\n#### Affected Actions:\n"
+    template += "- [ ] All\n"
+    template += "- [ ] Check Health\n"
+
+    actions = info.get("operations", [])
+    for action in actions:
+        template += f"- [ ] {action.get('title')}\n"
+    if not actions:
+        template += "_Add changes impact here_\n"
+    return template
+
+
+def add_unit_test_cases(template: str, info: dict) -> str:
+    template += "\n#### UTCs:\n"
+
+    template += \
+        "- [ ] Connector installation verified.\n" \
```

Comment:
Can we extract this list to a new constant class e.g. utc_constants.py and use here, so that we will get better control in selecting different set of UTCs, as needed.